ample/modelling/__main__.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ccp4-python
+
+__author__ = "Jens Thomas, and Felix Simkovic"
+__date__ = "22 Mar 2019"
+__version__ = "1.0"
+
+import argparse
+import os
+
+from ample.util import argparse_util
+from ample.util import logging_util
+from ample.modelling.rosetta_model import RosettaModel
+
+parser = argparse.ArgumentParser(description="AMPLE Modelling Module")
+argparse_util.add_core_options(parser)
+argparse_util.add_rosetta_options(parser)
+argparse_util.add_cluster_submit_options(parser)
+
+# Start logging to the console
+logging_util.setup_console_logging()
+
+# Get cmdline options or set defaults
+args = parser.parse_args()
+
+if args.rosetta_flagsfile is None:
+    raise RuntimeError("Need to supply a ROSETTA flagsfile with the -rosetta_flagsfile argument")
+flagsfile = args.rosetta_flagsfile
+if args.rosetta_AbinitioRelax is None:
+    raise RuntimeError("Need to supply a ROSETTA binary with the -rosetta_AbinitioRelax argument")
+flagsfile = os.path.abspath(args.rosetta_flagsfile)
+rosetta_binary = args.rosetta_AbinitioRelax
+
+nproc = args.nproc
+if nproc is None:
+    nproc = 1
+submit_cluster = args.submit_cluster
+if submit_cluster is None:
+    submit_cluster = True
+submit_qtype = args.submit_qtype
+if submit_qtype is None:
+    submit_qtype = 'SGE'
+submit_queue = args.submit_queue
+submit_array = args.submit_array
+if submit_array is None:
+    submit_array = True
+submit_max_array = args.submit_max_array
+
+
+nmodels = args.nmodels
+if nmodels is None:
+    nmodels = 1000
+work_dir = args.work_dir
+if work_dir is None:
+    work_dir = os.path.abspath('rosetta_modelling')
```

Comment:
Most of these if statements could be replaced by the default flag in argparse, unless of course there is a good reason not to set a default?